content/user/apps/Route-coverage.md

```diff
@@ -0,0 +1,40 @@
+<!--
+title: "Route Coverage"
+description: "Overview of setting up environments"
+tags: "user UI applications route coverage exercised vulnerabilities"
+-->
+
+## About Route Coverage
+
+An application “route” is a combination of three distinct data points: the normalized URL of the route, the HTTP verb associated with the request (e.g., Get or Post) and the method signature of the controller action. When you consistently exercising each route in your application, the Contrast agent can successfully Assess and Protect the surface layer of your application, and discover vulnerabilities. 
+
+### Agent configuration 
+
+Contrast supports route coverage for the following frameworks: 
+
+* Java Spring Spring MVC 4.x 
+* .NET MVC (versions 4 and 5), WebForms, Web API and WCF
+* Node Express 
+* Ruby Rails and Sinatra
+* Python, Django, Pyramid and Flask
```

Comment:
I think this should be something like Python: Django, Flask, and Pyramid.

Python is the language and the other three are frameworks.